packages/upload-client/src/defaultSettings.ts

```diff
@@ -9,11 +9,11 @@ const defaultSettings: DefaultSettings = {
   baseURL: 'https://upload.uploadcare.com',
   maxContentLength: 50 * 1024 * 1024, // 50 MB
   retryThrottledRequestMaxTimes: 1,
+  retryNetworkErrorMaxTimes: 3,
   multipartMinFileSize: 25 * 1024 * 1024, // 25 MB
   multipartChunkSize: 5 * 1024 * 1024, // 5 MB
   multipartMinLastPartSize: 1024 * 1024, // 1MB
   maxConcurrentRequests: 4,
```

Comment:
Can somebody use `multipartMaxAttempts`?

Reply:
@FilippovAM Yep. This is breaking change and we will release major version, see commit https://github.com/uploadcare/uploadcare-js-api-clients/pull/424/commits/178436e83c956a2b428e11bc4d76af27b627bcdb